keyboards/tiger910/keymaps/default/config.h

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
This file should be moved to `keyboards/tiger910/config.h`

Reply:
The two versions have different RGB number settings